recipes/boost/all/conanfile.py

```diff
@@ -251,7 +251,27 @@ def config_options(self):
     def _configure_options(self):
         return self._dependencies["configure_options"]
 
+    @property
+    def _fPIC(self):
+        try:
+            return self.options.fPIC
+        except ConanException:
+            return False
```

Comment:
It's deleted on Unix shared

```suggestion
            return True
```